packages/source-http/src/index.ts

```diff
@@ -64,8 +69,9 @@ export class SourceHttp implements Source {
 
   async fetch(offset: number, length?: number): Promise<ArrayBuffer> {
     try {
-      const Range = ContentRange.toRange(offset, length);
-      const headers = { Range };
+      const Range = {'range': ContentRange.toRange(offset, length)};
+      const headers = {...Range, ...this.headers};
```

Comment:
Can skip one of the `...`'s

```suggestion
      const headers = { range: ContentRange.toRange(offset, length), ...this.headers};
```